input/examples/condition-ckd.xml

```diff
@@ -3,9 +3,6 @@
   <!--  Example adapted from MCC eCare Plan Implementation Guide ConditionCKD  -->
```

Comment:
Remove <!--  Example adapted from MCC eCare Plan Implementation Guide ConditionCKD  --> this now irrelevant, and also given the changes no longer true.

Reply:
Removed